client/web/web.go

```diff
@@ -1121,21 +1268,6 @@ func (s *Server) proxyRequestToLocalAPI(w http.ResponseWriter, r *http.Request)
 	}
 }
 
-// localapiAllowlist is an allowlist of localapi endpoints the
-// web client is allowed to proxy to the client's localapi.
-//
-// Rather than exposing all localapi endpoints over the proxy,
-// this limits to just the ones actually used from the web
-// client frontend.
-var localapiAllowlist = []string{
```

Comment:
Removed the list here in favor of explicitly defining each of the supported endpoints as case statements in the `serveAPI` switch above.